build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.0.1'
+        classpath 'com.android.tools.build:gradle:4.1.0-rc02'
```

Comment:
Are we intending to use a release candidate here?

Reply:
Yeah Android Studio gradle autocomplete said this was the latest. I'm hoping it changes but it might be how Google's versioning releases now? 🤷‍♂️